### DISCUSSION TITLE
Further revisions to the AssetBackedSecurities ontology are needed

### DISCUSSION THREAD

**Discussion by ElisaKendall** [edmcouncil/fibo#1852] [Business Entities (BE)]
There are a number of definitions that need further work in the pool-backed securities ontology, and duplicate concepts should be eliminated.  In addition, some of the concepts in the asset-backed securities ontology also need work, per Pete's notes against SEC-163:

1. AutoLoanAssetBackedSecurity - I think it's just AutoAssetBackedSecurity. As per https://content.naic.org/sites/default/files/capital-markets-primer-consumer-abs.pdf . Since they are not necessarily loans but could be leases; I don't think it matters who issues them (the definition limits who can do so); As per the link in previous comment do we also want ConsumerABS as a superclass of this, CreditCardABS and SudentLoanABS?; what does this note mean "not shown"? The definition does correctly include "or leases"

2. AutoLoanPool - why not include leases and make the name of the class more general e.g. AutoFinancingPool

3. CashAssetBackedSecurity - what does this restriction mean in the real real world? If there's a subordination relationship that probably represents a contract that should be reified; the definition contradicts earlier declaration of AutoLoanABS as a subclass of this! Also the first example below is backed by real estate (HEL) and needs deconstructing

4. CreditCardAssetBackedSecurity - what granularity is a "receivable" - a single charge on a card? A monthly "amount due"? What if half the amount is paid off in the first month?
Can different receivables on the same card be in different pools?
How is this linked to actual credit card account in FIBO?

5. EsotericAssetBackedSecurity - "Promised" is redundant here - how is it any more promised than any other cash flow associated with an instrument?

6. HomeEquityLineOfCreditPool - we call them HELs earlier - are they the same thing?

7. hasUnderlyingCashFlow - hasUnderlyingCashFlow should be a super property you can apply to ANY debt instrument. Esoterics don't in fact nee a specific subProperty.

And from other discussion in the FCT, the classes that are specific to Bonds should be moved to a BondsExtension ontology temporarily, and ultimately merged into the main bonds ontology.

**Comment by mereolog** [comment]
@ElisaKendall Is it likely that we will close this issue soon? (It is quite old now)
Perhaps it should be transferred to https://github.com/edmcouncil/fibo/discussions?
